build.gradle.kts

```diff
@@ -1,6 +1,9 @@
 plugins {
+    @Suppress("DSL_SCOPE_VIOLATION")
```

Comment:
Can you add this comment to tell readers why we need this? 👀 
// https://youtrack.jetbrains.com/issue/KTIJ-19369/False-positive-cant-be-called-in-this-context-by-implicit-receiver-with-plugins-in-Gradle-version-catalogs-as-a-TOML-file#focus=Comments-27-6204464.0-0

Reply:
https://github.com/DroidKaigi/conference-app-2022/pull/502/commits/231a5830dad6932487a777594b8f438119b84873 Sure. I did.:+1: